environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/aws/AwsDiskEncryptionParameters.java

```diff
@@ -0,0 +1,72 @@
+package com.sequenceiq.environment.api.v1.environment.model.request.aws;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.Pattern;
+import com.google.common.annotations.VisibleForTesting;
+
+import com.sequenceiq.environment.api.doc.environment.EnvironmentModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "AwsDiskEncryptionV1Parameters")
+public class AwsDiskEncryptionParameters implements Serializable {
```

Comment:
Once #11024 gets merged, please pull & rebase so that these changes can disappear here. Furthermore, please be sure to squash your commits into a single one.